### ISSUE TITLE
Creating own templates of QR CODES

### ISSUE THREAD

**Issue by RousseauFlorian** [id1945/ngx-qrcode-styling#8]
Hello,

I want to create own template of QRCODES, can we have access to source code or there is a different method to create template.
I talk about DotType.

Thanks for you responding.

**Comment by id1945** [comment]
@RousseauFlorian You need to provide me with a sample. I'll put it in this library

**Comment by RousseauFlorian** [comment]
Can i make private assets ? Can you gavé me a sample ?

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by id1945** [comment]
@RousseauFlorian If you want to include an svg frame in this library then please provide me that svg file. I will add it to the repository and you can use it.

**Comment by RousseauFlorian** [comment]
No, it’s not the QRCODE frame that I want to change but the patterns on the QRCODE (like rounded,dots, classy, classy-rounded, square, extra-rounded). Is that possible?

**Comment by RousseauFlorian** [comment]
@id1945 Do you have a discord for have a better discussion ?

**Comment by id1945** [comment]
@RousseauFlorian Of course it can be changed.
![image](https://github.com/user-attachments/assets/1a700c75-97af-4637-bf93-628f4b350f4f)

**Comment by RousseauFlorian** [comment]
Yes, but custom dots (another of rounded,dots, classy, classy-rounded, square, extra-rounded) or another square (dots or corner like dot, square, extra-rounded) cannot be added if only we have source code, yes ?
So my question is: How can we add new shapes from the point and corner options other than those already included by you in the library?

**Comment by RousseauFlorian** [comment]
Up ???

**Comment by RousseauFlorian** [comment]
Pls can i have up ???
